doc/source/ray-core/api/core.rst

```diff
@@ -9,6 +9,7 @@ Core API
     ray.shutdown
     ray.is_initialized
     ray.job_config.JobConfig
+    ray.LoggingConfig
```

Comment:
This is the solution for https://github.com/ray-project/ray/pull/45344#discussion_r1625140344